packages/x-charts/src/internals/plugins/models/seriesConfig/tooltipGetter.types.ts

```diff
@@ -0,0 +1,12 @@
+import type { ItemTooltip } from '../../../../ChartsTooltip';
+import type {
+  ChartItemIdentifier,
+  ChartSeriesDefaultized,
+  ChartSeriesType,
+} from '../../../../models/seriesType/config';
+
+export type TooltipGetter<T extends ChartSeriesType> = (params: {
+  series: ChartSeriesDefaultized<T>;
+  getColor: (dataIndex: number) => string;
+  item: ChartItemIdentifier<T> | null;
```

Comment:
Maybe `identifier` or `itemIdentifier` woudl make more sens. From `item` we could expect to gat any value related to the item